src/VASL/build/module/map/QC.java

```diff
@@ -1416,8 +1416,12 @@ public void setup(boolean bln) {
       }
 
       if (sniperFinderButton != null && sniperFinderButton.getActionListeners().length == 0) {
-        sniperFinderButton.setIcon(new ImageIcon(Op.load("sniper").getImage()));
+        try {
+          sniperFinderButton.setIcon(new ImageIcon(Op.load("sniper").getImage()));
+        }
+        catch (NullPointerException ignored) {

```

Comment:
You should *never* catch NPEs. The right thing to do here is to check whether `Op.load("sniper").getImage()` returns `null`.